changelog/generated/v1.9.1.md

```diff
@@ -0,0 +1,7 @@
+## v1.9.1
+
+### Bug Fixes
+
+- Fixed the `operator-sdk generate bundle` command to specify the right path of bundle metadata in bundle.Dcokerfile. ([#5035](https://github.com/operator-framework/operator-sdk/pull/5035))
+- For Helm-based operators, fixed release equality comparison such that number values are compared and not their types to avoid unnecessary reconciliations. ([#5046](https://github.com/operator-framework/operator-sdk/pull/5046))
```

Comment:
The problem this fix attempts to resolve may still exist, and the original fix may be reverted. See https://github.com/operator-framework/operator-sdk/pull/5059#issuecomment-884336514. We may want to remove this change from v1.9.x until it gets resolved, so we don't have to do 2 reversions.

Reply:
On second thought, lets stick with the order of things in master.

Reply:
ok

Reply:
@estroz Didn't get your point. Do we need to remove change for helm #5046 ?